CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Next Release
 
 ### Breaking Changes
+- Java 21 is required to build and run Besu.
```

Comment:
I would say : Java 21 is required to build Besu, and at least Java 21 to run it.

Reply:
👍🏼